website_sale_vat_required/static/src/js/website_sale.tour.js

```diff
@@ -1,23 +0,0 @@
-/* Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
- * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
-
-odoo.define('website_sale_vat_required.tour', function (require) {
```

Comment:
If you remove this patch, then the website_sale main tour fails (as it's happening right now)